packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -138,9 +137,9 @@ export class Summarizer extends EventEmitter implements ISummarizer {
 
     public async run(
         onBehalfOf: string,
-        options?: Readonly<Partial<ISummarizerOptions>>): Promise<SummarizerStopReason> {
+        disableHeuristics?: boolean): Promise<SummarizerStopReason> {
```

Comment:
This class has access to this.configurationGetter(), so it should get state from there and not duplicate arguments - there should be one way to get to truth.

Reply:
please see my previous comments.